node/process.ts

```diff
@@ -1,4 +1,5 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// Copyright Joyent, Inc. and Node.js contributors. All rights reserved. MIT license.
```

Comment:
Added node.js license attribution because I copied some documentation for hrtime.